esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -429,10 +432,31 @@ EspHttpBinding::EspHttpBinding(IPropertyTree* tree, const char *bindname, const
                             }
                         }
 
-                        m_secmgr.setown(SecLoader::loadSecManager("LdapSecurity", "EspHttpBinding", LINK(lscfg)));
-                        if(m_secmgr.get() == NULL)
+                        StringBuffer resourcesBaseDnBuf;
+                        authcfg->getProp("@resourcesBasedn", resourcesBaseDnBuf);
+
+                        StringBuffer workunitsBaseDnBuff;
+                        authcfg->getProp("@workunitsBasedn", workunitsBaseDnBuff);
+
+                        std::string instanceParams;
+                        instanceParams.append(resourcesBaseDnBuf.str()).append(":").append(workunitsBaseDnBuff.str());
```

Comment:
I'm pretty sure that using the resources and workunits BaseDn should be enough to uniquely identify the secmgr we want. Given your knowledge and experience with what is _allowed_ to be configured and what is _conventionally_ configured could we have a clash in some rare usage? I don't recall if there even could be multiple servers configured in one process. Would we need something else to ensure uniqueness like LDAP server hostname or IP, perhaps?